test/e2e/tests/backup.spec.js

```diff
@@ -0,0 +1,82 @@
+const { strict: assert } = require('assert');
+const { promises: fs } = require('fs');
+const { convertToHexValue, withFixtures } = require('../helpers');
+
+const downloadsFolder = `${process.cwd()}/test-artifacts/downloads`;
+
+const createDownloadFolder = async () => {
```

Comment:
Can we reuse this method as we now have this in both the state-logs.spec.js and now backup.spec.js